xml/sle_update_preparation.xml

```diff
@@ -66,23 +66,23 @@
   </para>
 
   <para>
-   Check the release notes to see whether:
+   Check the release notes to check the following.
   </para>
 
   <itemizedlist>
    <listitem>
     <para>
-     your hardware needs special considerations;
+     Your hardware needs special considerations.
```

Comment:
leave as is

Reply:
Since when do we (or anyone else for that matter) end list items with `;` ?

Reply:
`;` are used to separate equally ranked parts of a sentence. Not particularly in lists though. It's not covered by our Style Guide. The `.` however is: Do not end list entries with a `.` if they consist of a single sentence only, see https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-list